src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write your code here
+const items = document.querySelectorAll('.population');
+
+const texts = [...items].map(item => item.innerText);
+const arr = [];
+
+for (const num of texts) {
+  const ojoj = num.replace(/[\s.,%]/g, '');
+
+  arr.push(Number(ojoj));
+}
+
+const total = arr.reduce((sum, el) => sum + el, 0);
+const average = arr.reduce((sum, el) => sum + el / arr.length, 0.0);
+
+const x = document.querySelector('.total-population');
+
+x.innerHTML = total.toLocaleString('de-DE');
+
+const y = document.querySelector('.average-population');
+
+y.innerHTML = average.toLocaleString('de-DE');
```

Comment:
please name variables according to what they mean, x, y are not understandable names